net.tangysoft.usenext.yaml

```diff
@@ -0,0 +1,64 @@
+app-id: net.tangysoft.usenext
+runtime: org.gnome.Platform
+sdk: org.gnome.Sdk
+runtime-version: '46'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.mono6
+rename-desktop-file: usenext.desktop
+command: usenext
+finish-args:
+  - --share=network
+  - --socket=x11
+  - --share=ipc
+  - --filesystem=home
+  - --env=MONO_MANAGED_WATCHER=dummy
+build-options:
+  append-path: /usr/lib/sdk/mono6/bin
+  append-ld-library-path: /usr/lib/sdk/mono6/lib
+  env:
+    PKG_CONFIG_PATH: /app/lib/pkgconfig:/app/share/pkgconfig:/usr/lib/pkgconfig:/usr/share/pkgconfig:/usr/lib/sdk/mono6/lib/pkgconfig
+modules:
+  - shared-modules/gtk2/gtk2.json
+
+  - name: mono-runtime
+    buildsystem: simple
+    build-options:
+      no-debuginfo: true
+      strip: true
+    build-commands:
+      - /usr/lib/sdk/mono6/install.sh
+
+  - name: gtk-sharp
+    sources:
+      - type: git
+        url: https://github.com/mono/gtk-sharp.git
+        commit: 3440993e00c89791c5401ae0dcc7508d81347739
+      - type: patch
+        path: gtk-sharp2-2.12.12-glib-include.patch
+      - type: shell
+        commands:
+          - ./bootstrap-2.12 --prefix=/usr
+
+  - name: usenext
+    buildsystem: simple
+
+    build-commands:
+      - ls -la
+      - ar x TangySoftSetup.deb
+      - tar xvf data.tar.gz
+      - mkdir -p /app/bin
+      - cp -ar usr/*  /app/
+      - sed -i 's,/usr/,/app/,g' /app/bin/usenext
+      - find .
+
+    post-install:
+      - install -t ${FLATPAK_DEST}/share/appdata/ -Dm644   net.tangysoft.usenext.metainfo.xml
+      - install -D usr/share/pixmaps/usenext.png  ${FLATPAK_DEST}/share/icons/${FLATPAK_ID}.png
```

Comment:
this path is incorrect.

Reply:
what is the correct path?
/app/share/icons/hicolor/$size/apps or /app/share/icons/hicolor/scalable/apps ?

Reply:
what the size of that PNG? I don't know. Definitely not scalable.